images/macos/toolsets/toolset-10.15.json

```diff
@@ -304,5 +304,11 @@
         "xcpretty",
         "bundler",
         "fastlane"
-    ]
+    ],
+    "default_ruby": {
```

Comment:
I think it should be called like `ruby` and contains `rubygems` and `default` like `java` section